From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Philipp Hancke <philipp.hancke@googlemail.com>
Date: Mon, 12 Oct 2026 14:53:43 +0200
Subject: [PATCH 09/21] Disable TLS session tickets for DTLS

They make no sense for WebRTC and increase the size of the last handshake
flight considerably. Same as
  https://webrtc-review.googlesource.com/c/src/+/362526
---
 src/aiortc/rtcdtlstransport.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/aiortc/rtcdtlstransport.py b/src/aiortc/rtcdtlstransport.py
index 553f5e0..00aabff 100644
--- a/src/aiortc/rtcdtlstransport.py
+++ b/src/aiortc/rtcdtlstransport.py
@@ -195,6 +195,7 @@ class RTCCertificate:
         self, srtp_profiles: list[SRTPProtectionProfile]
     ) -> SSL.Context:
         ctx = SSL.Context(SSL.DTLS_METHOD)
+        ctx.set_options(SSL.OP_NO_TICKET)
         ctx.set_verify(
             SSL.VERIFY_PEER | SSL.VERIFY_FAIL_IF_NO_PEER_CERT, lambda *args: True
         )
